docs/guides/src/main/operator/installation.adoc

```diff
@@ -5,14 +5,48 @@
 
 <@tmpl.guide
 title="Keycloak Operator Installation"
-summary="How to install the Keycloak Operator on Kubernetes">
-
-TODO: https://github.com/keycloak/keycloak/issues/10786
+summary="How to install the Keycloak Operator on Kubernetes and OpenShift">
 
 == Keycloak Operator Installation
+In this guide we will show how to install the Keycloak Operator in your Kubernetes or OpenShift cluster.
+
+=== OLM Installation
+
+You can install the Keycloak Operator in Kubernetes environment is to use Operator Lifecycle Manager (OLM).
+Before moving on, make sure you have OLM installed, if not follow this https://github.com/operator-framework/operator-lifecycle-manager/blob/master/doc/install/install.md#install-a-release[guide].
+
+To install the Keycloak Operator follow the https://olm.operatorframework.io/docs/tasks/install-operator-with-olm/[instructions].
+In the default Catalogs the Keycloak Operator is named `keycloak-operator` and make sure to use the `candidate` channel.
```

Comment:
```suggestion
In the default Catalog, the Keycloak Operator is named `keycloak-operator`. Make sure to use the `candidate` channel to find the operator.
```

Reply:
Why do we need to use the "candidate" channel? If this is a temporary thing, perhaps "to find the keycloak operator candidate release."?

Reply:
The `candidate` channel is currently required to comply with the [recommended naming](https://olm.operatorframework.io/docs/best-practices/channel-naming/#recommended-channel-naming). We will probably move to `stable` in KC 20.